launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.html

```diff
@@ -21,13 +21,14 @@ <h2>LaunchPod</h2>
     </mat-sidenav>
     <mat-sidenav-content>
         <mat-toolbar color="white">
-            <button type="button" aria-label="Toggle sidenav" mat-icon-button (click)="drawer.toggle()">
+            <button type="button" aria-label="Toggle sidenav" mat-icon-button (click)="drawer.open()">
```

Comment:
Just for my understanding, why did you change this from toggle to open? Did you want the button to only ever open and not also close? (Sorry, if I'm misunderstanding)

Reply:
The toggle was for before when our content would be pushed to 'side,' but now with the 'over' property clicking outside of the sidenav closes it automatically so we only require of drawer.open property